docs/advanced/Fastfile.md

```diff
@@ -114,6 +114,25 @@ end
 
 This will also automatically import all the local actions from this repo.
 
+
+## `import_from_gem`
+
+Import from another ruby gem, which you can use to create a single package with common `Fastfile`s and actions for all your projects.
+
+```ruby
+import_from_gem(gem_name: 'my_gem')
+# or
+import_from_gem(gem_name: 'my_gem',
+               paths: ['fastlane/Fastfile', 'fastlane/Fastfile_*'])
```

Comment:
uh weird indentation? 👀 Can we do e.g.:

```suggestion
import_from_gem(gem_name: 'my_gem', paths: ['fastlane/Fastfile', 'fastlane/Fastfile_*'])
```

or

```suggestion
import_from_gem(
  gem_name: 'my_gem',
  paths: ['fastlane/Fastfile', 'fastlane/Fastfile_*'],
)
```

or

```suggestion
import_from_gem(gem_name: 'my_gem', paths: [
  'fastlane/Fastfile',
  'fastlane/Fastfile_*',
])
```

? 🙈

Reply:
The intended indentation might've been this:

```suggestion
import_from_gem(gem_name: 'my_gem',
                paths: ['fastlane/Fastfile', 'fastlane/Fastfile_*'])
```

But I'm not a fan, hence why I suggested the 3 others above 😂 I'd pick the first one because I don't care much about horizontal line length, but the others are fine too